client/src/components/SelectLocationFormat.js

```diff
@@ -0,0 +1,41 @@
+import styled from "@emotion/styled"
+import { Location } from "models"
+import PropTypes from "prop-types"
+import React from "react"
+
+const SelectLocationFormat = ({ locationFormat, setLocationFormat }) => {
+  return (
+    <LocationFormatSelector
+      value={locationFormat}
+      name="locationFormat"
+      id="locationFormat"
+      onChange={e => {
+        setLocationFormat(e.target.value)
+      }}
+    >
+      {Object.values(Location.LOCATION_FORMATS).map(format => (
+        <option value={format} key={format}>
+          {format}
+        </option>
+      ))}
+    </LocationFormatSelector>
+  )
+}
+
+SelectLocationFormat.propTypes = {
+  locationFormat: PropTypes.string.isRequired,
+  setLocationFormat: PropTypes.func.isRequired
+}
+
+const LocationFormatSelector = styled.select`
+  margin-left: 10px;
+  outline: 2px solid #337ab7;
+  font-size: 14px;
+  margin-left: 1rem;
```

Comment:
`margin-left` is set twice. Also I suggest removing `outline` and add `padding: 4px` which creates a better view in both Chrome and Firefox